### ISSUE TITLE
プッシュ通知の日時指定を複数設定する方法について

### ISSUE THREAD

**Issue by TaigaTakeshita** [NIFCLOUD-mbaas/UserCommunity#1336]
プッシュ通知の日時指定について質問があります。
mbaasの管理画面からのプッシュ通知発信には時間指定機能がありますが、
アプリ側からプッシュ通知の時間指定を行うことは可能でしょうか？

また、1日前・１時間前・30分前といったように複数設定することは可能でしょうか？
ご教示下さると幸いです。

**Comment by goofmint** [comment]
> アプリ側からプッシュ通知の時間指定を行うことは可能でしょうか？

はい、できます。Swift SDKであればdeliverTimeで指定します。他のSDKでも似たようなメソッドがあるはずです。

https://nifcloud-mbaas.github.io/ncmb_swift/Classes/NCMBPush.html#/s:4NCMB8NCMBPushC12deliveryTime10Foundation4DateVSgvp

> また、1日前・１時間前・30分前といったように複数設定することは可能でしょうか？

こちらは意図が読みかねていますが、複数のプッシュ通知を作成すれば良いのかなと。

**Comment by TaigaTakeshita** [comment]
@goofmint 
> > アプリ側からプッシュ通知の時間指定を行うことは可能でしょうか？
> 
> はい、できます。Swift SDKであればdeliverTimeで指定します。他のSDKでも似たようなメソッドがあるはずです。

ご回答いただきありがとうございます。
可能なのですね。
ちなみにJavaScriptを用いて開発を行っているのですが、Swift SDKのdeliverTimeに類似したメソッドをご存知でしたら教えていただけないでしょうか？

> 
> > また、1日前・１時間前・30分前といったように複数設定することは可能でしょうか？
> 
> こちらは意図が読みかねていますが、複数のプッシュ通知を作成すれば良いのかなと。
おっしゃる通り、複数プッシュを作成すれば良いですね。
アドバイスありがとうございます。

**Comment by goofmint** [comment]
`deliveryTime` で指定します。

https://mbaas.nifcloud.com/doc/current/rest/push/pushRegistration.html

設定は他のパラメーターと同じです。

https://mbaas.nifcloud.com/doc/current/push/basic_usage_javascript.html#SDK%E3%82%92%E5%88%A9%E7%94%A8%E3%81%97%E3%81%A6%E3%83%97%E3%83%83%E3%82%B7%E3%83%A5%E9%80%9A%E7%9F%A5%E3%82%92%E9%85%8D%E4%BF%A1%E3%81%99%E3%82%8B
